src/Signup.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+const Signup = (): JSX.Element => {
+  const [name, setName] = useState("");
```

Comment:
These inputs can be handled by one useState, [here](https://dev.to/deboragaleano/how-to-handle-multiple-inputs-in-react-55el) you can read more about it.